### PR TITLE
Bump Bio-Formats version to 5.1.3-SNAPSHOT

### PR DESCRIPTION
**Pull request by joshmoore**
Should now be built locally in the metadata-ci
build system.

### CODE REVIEW

File: etc/omero.properties [hunk 1]
```diff
@@ -895,7 +895,7 @@ versions.asm=1.5.3
 versions.axis=1.4
 versions.backport=3.1
 versions.batik=1.8pre-jdk6
-versions.bioformats=5.1.4
```
Comment (mtbc) [inline etc/omero.properties]: not 5.1.7?

Comment (joshmoore) [inline etc/omero.properties]: Eventually, yes, but currently the metadata branch is behind. I'm attempting to test openmicroscopy/bioformats#2227 in isolation. With the bump of devspace, metadata-ci hasn't been using the merge build for several days, which means that `bin/omero import -f $TARA` currently lists "UNKNOWN_FORMAT". Once that's working again, I'll look into bumping the bioformats/metadata branch to include 5.1.7 or optimally 5.1.8.

Comment (mtbc) [inline etc/omero.properties]: Thank you for explaining. :+1:

**Comment by joshmoore** [comment]
--rebased-to #4465

**Comment by joshmoore** [comment]
Closing to let gh-4506 in. Afterwards, we can merge 5.1.8 into the metadata branch and continue using "5.1.8-SNAPSHOT".

**Pull request closed without merging**